docs/deployment/digital-asset.md

```diff
@@ -379,23 +379,37 @@ await lspFactory.LSP8IdentifiableDigitalAsset.deploy({...}, {
 
 ### IPFS Upload Options
 
-You can specify how you want your profile metadata to be uploaded by passing the `ipfsClientOptions` inside the `options` object. Here you can set the IPFS gateway where you want the metadata to be uploaded.
+You can specify how you want your profile metadata to be uploaded by passing the `ipfsGateway` inside the `options` object. Here you can set the IPFS gateway where you want the metadata to be uploaded.
```

Comment:
**Suggested change:**

You can specify the IPFS network where you want to upload your profile metadata using an `ipfsGateway` object inside the `options` parameter.